renderer-macos/lib/stubs/NSTextView_stubs.h

```diff
@@ -1,11 +1,34 @@
 #import "Cocoa_stubs.h"
 #define CAML_NAME_SPACE
-#import <caml/mlvalues.h>
+#import <caml/alloc.h>
 
 @interface TextView : NSTextView
```

Comment:
Keep in mind that there's no name spacing in Objective-C. That's why class names should in general be prefixed with a two or three letter prefix.

Reply:
`BR` everywhere? `BRApplicationDelegate`, `BRView`, `BRButton`, `BRTextView`.

Reply:
I know you're gonna say it's gross, but I'd say `BRISKView` etc. BR is too common.

Reply:
Can we at least make it `BriskView`?

Reply:
yes we can. BRISKView looks very objcy though and I dig it 😄

Reply:
I'm not sure if we should rename `NSView_stubs` to `BriskView`, if the former makes perfect sense.